app/scripts/HiGlassComponent.js

```diff
@@ -3209,31 +3187,23 @@ class HiGlassComponent extends React.Component {
         this.yScales[v.uid] = scaleLinear().domain(v.initialYDomain);
       }
 
-      // Add names to all the tracks
-      let looseTracks = positionedTracksToAllTracks(v.tracks);
-
-      // give tracks their default names (e.g. 'type': 'top-axis'
-      // will get a name of 'Top Axis'
-      looseTracks = this.addUidsToTracks(looseTracks);
```

Comment:
Are you sure this can be removed? Doesn't this add uids to tracks that don't have them?

Reply:
Yes because `TiledPlot` does the exact same thing again. Look at line 81.

Or am I missing something?